alma/controllers/hook/DisplayRefundsHookController.php

```diff
@@ -113,11 +113,8 @@ public function run($params)
         $wording = [
             'title' => $this->module->l('Alma refund', 'DisplayRefundsHookController'),
             'description' => sprintf(
-                $this->module->l(
-                    // phpcs:ignore Generic.Files.LineLength
-                    'Refund this order thanks to the Alma module. This will be applied in your Alma dashboard automatically. The maximum refundable amount includes client fees. %1$sSee documentation%2$s',
-                    'DisplayRefundsHookController'
-                ),
+                // phpcs:ignore Generic.Files.LineLength
```

Comment:
Is this `// phpc:ignore` comment always useful ?

I think we has updated your phpcs.xml file to allow long lines and you could now remove this kind of comments (if you continue to have issue with phpcs, I think you could use the value we are using [in sonar](https://sonarcloud.io/organizations/almapay/rules?activation=true&q=long&qprofile=AYN5n3WymsMwGzYL91fv&types=CODE_SMELL&open=php%3AS103) for  prestashop => `520`

Reply:
@syjust-alma Yes for me we had always the limit.
I saw in Sonar I see always 120 in limit
<img width="344" alt="Capture d’écran 2022-11-23 à 10 08 17" src="https://user-images.githubusercontent.com/89775252/203508327-38c0deef-fa80-482b-8c6d-cf71d647fd8a.png">

Reply:
There is a custom rules dedicated to PrestaShop. Let's discuss about that today ;)

Reply:
It's done @syjust-alma 
I merge the PR.
Thanks